test/plugins/features/movies/controller.test.js

```diff
@@ -1,11 +1,51 @@
 'use strict';
 
+const Bluebird   = require('bluebird');
 const Controller = require('../../../../lib/plugins/features/movies/controller');
-const Movie      = require('../../../../lib/models/movie');
 const Knex       = require('../../../../lib/libraries/knex');
+const Location   = require('../../../../lib/models/location');
+const Movie      = require('../../../../lib/models/movie');
+
+const movie_list = [
```

Comment:
nit: can you make `movie_list` and `locations_list` all uppercase (i.e. `MOVIE_LIST` and `LOCATION_LIST`) since they're effectively constants? also to make them consistent, either make `movie` plural or `locations` singular